src/Text/Text.js

```diff
@@ -1,64 +0,0 @@
-import { css } from '@emotion/react';
-import styled from '@emotion/styled';
-import {
-  color,
-  fontSize,
-  fontWeight,
-  lineHeight,
-  maxWidth,
-  minWidth,
-  overflow,
-  space,
-  textAlign
-} from 'styled-system';
-
-import textTransform from '../utils/textTransform';
-import textOverflow from '../utils/textOverflow';
-import whiteSpace from '../utils/whiteSpace';
-
-const baseStyles = ({ theme }) => css`
-  font-family: ${theme.brandFont};
-`;
-
-const Text = styled.p`
-  ${baseStyles};
-
-  ${color};
-  ${fontSize};
-  ${fontWeight};
-  ${lineHeight};
-  ${maxWidth};
-  ${minWidth};
-  ${overflow};
-  ${space};
-  ${textAlign};
-  ${textTransform};
-  ${textOverflow};
-  ${whiteSpace};
-`;
-
-Text.propTypes = {
-  ...color.propTypes,
-  ...fontSize.propTypes,
-  ...fontWeight.propTypes,
-  ...lineHeight.propTypes,
-  ...maxWidth.propTypes,
-  ...minWidth.propTypes,
-  ...overflow.propTypes,
-  ...space.propTypes,
-  ...textAlign.propTypes,
-  ...textTransform.propTypes,
-  ...textOverflow.propTypes,
-  ...whiteSpace.propTypes
-};
-
-Text.defaultProps = {
-  color: 'text.default',
-  fontSize: 'size4',
-  lineHeight: 'small'
-};
-
-// Deprecated -- use <Text as="span"></Text>
-Text.Span = (props) => <Text as="span" {...props} />;
```

Comment:
I removed this from the TS version since it's been deprecated for ages.  I have a changeset for the main app ready to go that will handle the change.